frontend/src/global_styles/content/work_packages/single_view/_single_view.sass

```diff
@@ -51,6 +51,21 @@
       line-height: 1
 
 
+
+.work-package-tab--breadcrumb
```

Comment:
not sure if there is a file for the work-package-tab block, but if so this should be there